addons/test_mass_mailing/tests/test_mailing_sms.py

```diff
@@ -147,12 +153,19 @@ def test_mass_sms_internals_errors(self):
              for record in falsy_record_1 + falsy_record_2],
             mailing, falsy_record_1 + falsy_record_2,
         )
+        # check note posted on records
+        expected_message_counts = list(repeat(1, 10)) + [1, 0, 0, 0, 0]
```

Comment:
`list(repeat(1, 10))` can just be `[1] * 10` since we make it a list anyway
(same in other places)